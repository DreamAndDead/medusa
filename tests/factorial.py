def factorial(value):
    return 1 if value == 0 else value * factorial(value - 1)

print(factorial(5))
print(factorial(10))
print(factorial(3))
print(factorial(0))
