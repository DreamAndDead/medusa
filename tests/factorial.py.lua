dofile('./pythonlua/luainit.lua')

local function factorial(value)
    return (value == 0) and 1 or (value * factorial((value - 1)))
end
print(factorial(5))
print(factorial(10))
print(factorial(3))
print(factorial(0))