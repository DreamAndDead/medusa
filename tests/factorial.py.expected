120
3628800
6
1
